src/java.desktop/share/classes/java/awt/Cursor.java

```diff
@@ -300,16 +300,15 @@ public static Cursor getSystemCustomCursor(final String name)
             String prefix = CURSOR_DOT_PREFIX + name;
             String key    = prefix + DOT_FILE_SUFFIX;
 
-            if (!systemCustomCursorProperties.containsKey(key)) {
+            String fileName = systemCustomCursorProperties.getProperty(key);
```

Comment:
You can keep the `final` modifier, probably.

Reply:
I really like how this line now fits in 80 characters limit. But if you insist - I will return it.

Reply:
That's what I thought. At the same time, it's only two columns longer than 80 chars a line.

The usage of `final` modifiers seems inconsistent in the method. Maybe remove `final` from `localized` too? Then there would be no `final` modifiers till `hotPoint`.

Reply:
Remove more `final`s. Code _feels_ cleaner without them.